pattern/text/en/inflect.py

```diff
@@ -609,7 +612,12 @@ def singularize(word, pos=NOUN, custom={}):
             return word
     for x in singular_ie:
         if w.endswith(x+"s"):
-            return w
+            prefix = w[:w.find(x)]
+            return prefix+x
```

Comment:
1. I think it should have returned `word` and not `w`
2. Isn't `prefix = word[:w.find(x)]; return prefix+x` just a complex version of `return word[:-1]`?